.github/workflows/daily.yml

```diff
@@ -0,0 +1,70 @@
+name: Build and test
```

Comment:
perhaps call the file "nigthly" and here also a different name than "Build and test" (to not duplicate existing workflow name)

Reply:
Done